spec/unit/Environment_spec.lua

```diff
@@ -19,7 +19,7 @@ local function reset_websocket_stub(stub)
             publicKeyExponent = "010001"
         }
     end
-    stub.send_login_credentials = function() return {sessionId = "sessionId0"} end
+    stub.send_login_credentials = function() return {sessionId = 12345} end
```

Comment:
string.format fails when formatting a string with `%d` in the log message. Actual value from the database is a number